src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobBuilder.java

```diff
@@ -268,7 +268,8 @@ public boolean perform(final AbstractBuild<?, ? > build, final Launcher launcher
         final CounterManager phaseCounters = new CounterManager();
         boolean aggragatedTestResults = false;
         for (PhaseJobsConfig phaseJobConfig : phaseJobs) {
-            Item item = jenkins.getItem(phaseJobConfig.getJobName(), multiJobBuild.getParent(), AbstractProject.class);
+            String resolvedJobName = expandToken(phaseJobConfig.getJobName(), build, listener);

```

Comment:
Could you add a test for this, possibly in https://github.com/HBOCodeLabs/tikal-multijob-plugin/blob/enelson/expandjobname/src/test/java/com/tikal/jenkins/plugins/multijob/test/PhaseJobsConfigTest.java ?